gems/quilt_rails/test/quilt/header_csrf_strategy_test.rb

```diff
@@ -7,7 +7,7 @@ def test_raises_an_exception_if_the_samesite_header_is_missing
       DummyRequest.any_instance.stubs(:headers).returns({})
       strategy = HeaderCsrfStrategy.new(DummyController.new)
 
-      assert_raises HeaderCsrfStrategy::NoSameSiteHeaderError do
+      assert_raises(HeaderCsrfStrategy::NoSameSiteHeaderError) do
```

Comment:
was it complaining about missing parens?

Reply:
Yes